### DISCUSSION TITLE
Note Toolbar for sequential files NOT Daily Notes

### DISCUSSION THREAD

**Discussion by dlangendorf** [chrisgurney/obsidian-note-toolbar#107] [Q&A]
First of all, Chris, you're a Godsend. Thank you for all your hard work. I spread the Note Toolbar word as much as I can.

Getting to the idea/question, I use the toolbar for Daily Notes. I also create a daily health report so I don't clutter the formal daily note. Daily Note: [[2024-07-24]] ... Daily Health: [[2024-07-24 Daily Health]]. I just link the Daily Health file in that day's Daily Note. **Is it possible to create a Note Toolbar item for the day's daily health note?** If so, I'm not sure how to do it. I can create a toolbar button for the Daily Health folder, but no idea if I can go to that day's note.

Anyway, thought I'd ask. Again, thanks for all your work.

**Comment by chrisgurney** [comment]
@dlangendorf Thanks for the kind comments!

The solution here is a URI item with a [variable](https://github.com/chrisgurney/obsidian-note-toolbar/wiki/Variables), specifically `{{note_title}}`:

<img src="https://github.com/user-attachments/assets/ede5f877-5806-4c3e-9ac1-fd6bad0ee1ab" width="900"/>

Here's the configuration I used for that toolbar item:

<img src="https://github.com/user-attachments/assets/0f196a6a-13e1-4571-9b79-7cbaa9dbb3bb" width="450"/>

Notes:

- This assumes the _Daily Health_ note is in the same folder, but you should be able to add a folder to the URI if it resides elsewhere.
- If you want to create the health note on click (if it doesn't already exist), make sure you add the path to your Daily Notes folder to the URI, otherwise it will be created in the root of the vault. (I'm now realizing that it might be nice to have the path in a variable, so I'll add that to my todo list.)
- Using this technique, you will _not_ see a connection from the daily note to the health note in the graph view (if that matters to you), so you may still want to link it somewhere in the note. An alternative might be to link it from a property, and then reference the property from the URI, e.g., `{{prop_health}}`

Please let me know if this works for you!

**Reply by dlangendorf** [reply]
Chris, thanks so much. I'll give it a try soon and reach out if I mess something up -- very likely. I'll pass along other use scenarios if I think they're of any value to you and other users.

**Reply by chrisgurney** [reply]
@dlangendorf Thank you and good luck!

I'm curious, where do you think I should be advertising / talking about the plugin, to help spread the word?

**Reply by chrisgurney** [reply]
One other catch with the above option: The button stays in the toolbar on the health note, and doesn't link to anything useful at that point.

The other option to use a property might be a better bet, as the button will be removed automatically if the property does not exist.

Let me know if you need help setting this up.

**Reply by chrisgurney** [reply]
@dlangendorf Here's a demo of how to do it with the property:

![CleanShot 2024-07-25 at 08 50 17](https://github.com/user-attachments/assets/9ff0c560-7edc-4a72-9d85-480936f144fe)

**Comment by chrisgurney** [comment]
Hey @dlangendorf just checking in to see if you were able to give this a shot? If not, please me know how it goes!

**Comment by chrisgurney** [comment]
Hey @dlangendorf one last comment, now that Note Toolbar supports scripting you could:

1. Create a simple script that checks for the `Daily Health` note using the current file's filename, and open it if it exists.
2. Depending on the plugin you want to use, you could then execute it using Templater (via `Execute Templater file`), Dataview (`Execute JavaScript`), or JS Engine (`Execute JavaScript`).

(There would be no need for the note property, as required in the previous solution I mentioned.)

LMK if you'd like help with the above. Will close this thread for now.
